Dockerfile

```diff
@@ -1,5 +1,31 @@
+FROM rslota/beam-builder:erlang-22.0_elixir-1.9 AS builder
```

Comment:
Can we move the `beam-builder` docker image under `mongooseim` or `esl` org on dockerhub?

Reply:
Yeah, we can.

Reply:
But to avoid miscommunication - this is big enough in terms of waiting time, that definitely not as part of this PR, as it blocks quite a few things.